React/CoreModules/RCTWebSocketModule.mm

```diff
@@ -168,7 +168,11 @@ - (void)webSocket:(RCTSRWebSocket *)webSocket didFailWithError:(NSError *)error
   NSNumber *socketID = [webSocket reactTag];
   _contentHandlers[socketID] = nil;
   _sockets[socketID] = nil;
-  [self sendEventWithName:@"websocketFailed" body:@{@"message" : error.localizedDescription, @"id" : socketID}];
+  NSDictionary *body = @{
+        @"message" : error.localizedDescription ?: @"",
```

Comment:
Should this say something like "undefined" instead of empty string?

Reply:
Thank you for your comment. I updated the error message to `Undefined, error is nil`. 
Please let me know if you have any further feedback.